README.md

```diff
@@ -12,7 +12,12 @@ Thanks to Malcolm (Maxstang) for the boards, support, testing and encouragement.
 
```

Comment:
Please run the changes to readme.md through a markdown editor to ensure formatting changes are good.  If you change typos etc, that's great, but don't modify the formatting please.